compiler/src/dotty/tools/dotc/core/Mode.scala

```diff
@@ -124,4 +124,6 @@ object Mode {
    *  This mode forces expansion of inline calls in those positions even during typing.
    */
   val ForceInline: Mode = newMode(29, "ForceInline")
+
```

Comment:
Add comment explaining how this mode modifies subtyping to achieve the desired behaviour.

Reply:
Comments added